test/fixtures/dialects/snowflake/snowflake_create_view.sql

```diff
@@ -0,0 +1,42 @@
+
+create view another_view comment = 'a great description' as select col_1, col_2 from other_table;
+
+CREATE VIEW basic_view AS SELECT col1, col2 FROM src_table;
+
+CREATE VIEW view_with_comments
+COMMENT = 'my comment' AS SELECT col1, col2 FROM src_table;
+
+CREATE OR REPLACE VIEW view_with_replace_and_comment
+COMMENT = 'my comment' AS SELECT col1, col2 FROM src_table;
+
+CREATE OR REPLACE SECURE RECURSIVE VIEW IF NOT EXISTS secure_recursive_view_with_comment
+COMMENT = 'my comment' AS SELECT col1, col2 FROM src_table;
+
+CREATE OR REPLACE VIEW view_with_comment_and_copy_grants
+COMMENT = 'my comment'
+COPY GRANTS
+AS SELECT col1, col2 FROM src_table;
+
+CREATE OR REPLACE VIEW view_with_tags_and_copy_grants
+WITH TAG (foo = 'bar', hello = 'world')
+COPY GRANTS
+AS SELECT col1, col2 FROM src_table;
+
+CREATE OR REPLACE VIEW view_with_column_comment
+(
+    col1,
+    col2 COMMENT 'some comment'
+)
+AS SELECT col1, col2 FROM src_table;
+
+CREATE OR REPLACE SECURE RECURSIVE VIEW IF NOT EXISTS view_with_all_implemented_features
+    COMMENT = 'table-level comment'
+    (
+        col1,
+        col2 COMMENT 'some comment'
+    )
```

Comment:
Feels weird not defining the column list immediately after the view name. Could you confirm that this is valid syntax in snowflake?

Same question for the other options that we're allowing to parse before the column list?
e.g.
```sql
CREATE OR REPLACE VIEW IF NOT EXISTS view_with_all_implemented_features
    WITH TAG (foo = 'bar', hello = 'world')
    COPY GRANTS
    (
        col1,
        col2 COMMENT 'some comment'
    )
AS SELECT col1, col2 FROM src_table;
```

Reply:
I don't have access to the tag feature in my Snowflake instance, but I was able to verify that this syntax works.

```
CREATE OR REPLACE VIEW sandbox.test_view COMMENT = 'hello'
COPY GRANTS
(
  user_id, company_id COMMENT 'some comment'
)
AS SELECT id_user, id_workspace FROM sandbox.test_table;
```

Reply:
Nice thanks for confirming! 😄